plugins/CoreAdminHome/templates/generalSettings.twig

```diff
@@ -1,290 +1,226 @@
 {% extends 'admin.twig' %}
 
 {% block content %}
-{# load macros #}
-{% import 'macros.twig' as piwik %}
-{% import 'ajaxMacros.twig' as ajax %}
+    {% import 'macros.twig' as piwik %}
+    {% import 'ajaxMacros.twig' as ajax %}
 
-{% if isSuperUser %}
```

Comment:
Edit: please ignore (the change is actually correct as this template is only created in a controller action that already checks for Super User. )